addons/mail/static/src/js/services/mail_document_thread_manager.js

```diff
@@ -194,10 +202,17 @@ MailManager.include({
                 resModel: info[0],
             });
             if (state.windowState === 'closed') {
-                documentThread.close();
+                documentThread.close({
+                    ignoreCrossTabSync: true,
+                });
             } else {
-                documentThread.fold(state.windowState === 'folded');
-                self.openThreadWindow(documentThread.getID(), { keepFoldState: true });
+                documentThread.fold(state.windowState === 'folded', {
+                    ignoreCrossTabSync: true,
```

Comment:
It's debatable but, the keyworkd "ignoreCrossTabSync" makes me think the other way than it should.
It makes me think that the thing I'm calling will ignore any cross tab sync it may get but in fact it just doesn't make any cross tab sync. Maybe something like "performCrossTabSync" (with an inversion of true/false logic) or "skipCrossTabSync" (without inversion) would be more accurate. (It's not very important though)